src/discord.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const discordjs = require('discord.js')
+const getRepo = require('./parser').getRepoName
+
+var client = new discordjs.Client()
+var token = process.env[`npm_package_config_token`]
+var notify = process.env[`npm_package_config_discordnotify`]
+var default = process.env[`npm_package_config_discorddefault`]
+
+if (!notify) return;
+notify = JSON.parse(notify)
+
+if (token) {
+    client.login(token);
+}
+
+var loggedIn = false;
+
+client.on('ready', function() {
+    console.log('Successfully connected to Discord');
+    loggedIn = true;
+})
+
+if (!loggedIn) console.log('The bot has not successfully logged in to Discord.')
```

Comment:
Race with the `ready` handler

Reply:
I ended up changing this to notify only when and if a webhook is received and if at that time the bot is not yet logged in